README.md

```diff
@@ -709,9 +789,12 @@ An example is available [here](examples/android_groovy)
 </details>
 
 ## Implicit plugin dependencies
-While the plugin is being applied, the artifacts of the JaCoCo or Kover toolkit are dynamically loaded. They are downloaded from the `mavenCentral` repository.
 
-For the plugin to work correctly, you need to make sure that the `mavenCentral` (or any of its mirrors) is added to the repository list of the project in which the plugin is applied, if it doesn't already exist (usually this is the root project):
+While the plugin is being applied, the artifacts of the JaCoCo or Kover toolkit are dynamically loaded. They are 
```

Comment:
I don't quite understand what `While the plugin is being applied` means - do you mean when you've just added it or when you're running Kover? It sounds like the first, but then it seems kinda obvious that Kover would download dependencies and fail if it can't find an artifact?

Reply:
This is approximately both.
[Applying plugin](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) is a code like  
```
plugins {
    id("...")
}
```

The plugin is applied every time the build is run, even if no Kover command is explicitly invoked.
However, it is difficult to say unequivocally when exactly the dependencies are resolved - directly at the time of applying plug-in, or at the start of the Kover commands.

Perhaps it is better to reformulate: "during the running of the build ..."